app/Helpers/AirfieldMonitoringHelper.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+function monitoredAirfields() {
+    return [
+        //"AGS" => "Augusta",
+        "ATL" => "Atlanta",
+        //"AVL" => "Asheville",
+        //"BHM" => "Birmingham",
+        //"CHA" => "Chattanooga",
+        "CLT" => "Charlotte",
+        //"GSP" => "Greensboro",
+        //"MGM" => "Montgomery",
+        //"PDK" => "Peachtree",
+        //"TRI" => "Tri-Cities",
+        //"TYS" => "Tyson"
```

Comment:
Let's remove these if they need to be commented out. Or uncomment them if they should be checked as well.